packages/app/src/cli/api/graphql/all_orgs.ts

```diff
@@ -5,7 +5,8 @@ export interface AllOrganizationsQuerySchemaOrganization {
   businessName: string
   website: string
   betas: {
-    appUiDeployments: boolean
+    appUiDeployments?: boolean
+    cliTunnelAlternative?: boolean
```

Comment:
Do we need to retrieve the betas in all these graphql requests? It would be nice to remove it if it's not used in all the cases to avoid making them slower 🙏

Reply:
Done @gonzaloriestra, thanks for the feedback, much cleaner now 🙇